packages/frontend-shared/src/assets/icons/file-list.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="16" height="10" viewBox="0 0 16 10" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M1 7H3V9H1V7Z" fill="white" fill-opacity="0.4" class="icon-light"/>
```

Comment:
I think we would expect to have icons on a white background by default, so thesis would be more usable with `icon-dark` and some kind of dark gray.

Reply:
Changed both icons to use icon dark and have a default dark color.